src/DataManagement/Entries/EntryList.cs

```diff
@@ -17,13 +17,19 @@ public EntryList(EntryType entryType)
             EntryType = entryType;
         }
 
+        public EntryList(IEnumerable<IEntry> entryList)
+        {
+            if (entryList is null) throw new ArgumentException();
+            AddRange(entryList);
```

Comment:
Ofc I just could google, but what does this addrange do? Does it just adds everything it received as parameter into "itself"?

Reply:
Since the class is an extended List, it's just the same as AddRange for any other list, so in this case it justs add everything into itself, yes :D